models/qualtrics__survey.sql

```diff
@@ -16,7 +16,7 @@ question as (
     from {{ var('question') }}
 ),
 
--- should we bring in distribution_contact for like the number of currently pending or opened invites?
+-- should we bring in distribution_contact for like the number of currently pending or opened invites? 
```

Comment:
Similar comment around this being a good metric that I feel could add a lot of value, but not necessarily crucial in my opinion. If the LOE is relatively small then I say yes. If not, then I say we hold off for now.

Reply:
removing